renku/ui/service/controllers/versions_list.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 - Swiss Data Science Center (SDSC)
```

Comment:
If I am not mistaken the Copyright year should be kept up to date

Reply:
Ok I updated the copyright year of any file I touched with this PR. I am really not sure what the year there means or what the consequences of changing or not changing are.

@Panaetius let me know if you want me to do something different.

Reply:
no that's fine. I think it's mostly cargo cult since none of us are lawyers. In renku-python we have like `Copyright 2017-2022` but then we have no basis on judging what the right way to do it would be (or if file headers are even necessary next to the LICENSE file).